src/components/org/OrgExplorer.vue

```
@@ -115,16 +116,92 @@ const modeClass = computed(() => ({
+  if (node.id === nodeId) return nextPath
+  for (const child of node.children || []) {
+    const found = findPathByNodeId(child, nodeId, nextPath)
+    if (found) return found
+  }
+  return null
+}
+
+const collectAllNodeIds = (node, acc = {}) => {
+  if (!node) return acc
+  acc[node.id] = true
+  ;(node.children || []).forEach((child) => collectAllNodeIds(child, acc))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)